sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-blob/inc/azure/messaging/eventhubs/checkpointstore_blob/dll_import_export.hpp

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/**
+ * @file
+ * @brief DLL export macro.
+ */
+
+// For explanation, see the comment in azure/core/dll_import_export.hpp
+
+#pragma once
+
+/**
+ * @def AZ_MESSAGING_EVENTHUBS_DLLEXPORT
+ * @brief Applies DLL export attribute, when applicable.
+ * @note See https://docs.microsoft.com/cpp/cpp/dllexport-dllimport?view=msvc-160.
+ */
+
+#if defined(AZ_MESSAGING_EVENTHUBS_DLL) || (0 /*@AZ_MESSAGING_EVENTHUBS_DLL_INSTALLED_AS_PACKAGE@*/)
```

Comment:
I think, in this file you should rename `AZ_MESSAGING_EVENTHUBS` to `AZ_MESSAGING_EVENTHUBS_CHECKPOINTSTORE_BLOB`